Cargo.toml

```diff
@@ -107,3 +108,5 @@ debug = false
 [patch.crates-io]
 webpki = { git = "https://github.com/seanmonstar/webpki", branch = "cert-dns-names" }
 
+[dependencies.kafka_codec]
+path = "../kafka_codec"
```

Comment:
This points to the codebase of https://github.com/ruig2/KafkaRustCodec/tree/rui/add_metadata_decode